lib/ansible/modules/benchmarks/stress.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Hugh Ma <Hugh.Ma@flextronics.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: stress
+short_description: Execute the Stress benchmark
+description:
+ - Use this module to run the Stress benchmark
+ - U(https://people.seas.harvard.edu/~apw/stress/)
+version_added: "2.3"
+options:
+  chdir:
+    description:
+     - Change the current working directory
+    required: false
+    default: null
+  cpu:
+    description:
+     - The total number of CPU-bound processes for stress.
+    required: true
+  executable:
+    description:
+     - The path to the stress executable if running from source.
+    required: false
+  dest:
+    description:
+     - Absolute path of file to write/read stdout (results) to/from.
+    required: false
+  state:
+    description:
+     - Execute module to run stress or parse an existing stress result file.
+    required: true
```

Comment:
Please add the following so the documentation will list the acceptable choices.
`choices: ['benchmark', 'parse']`